owner/manager.go

```diff
@@ -228,6 +232,7 @@ func (m *ownerManager) campaignLoop(etcdSession *concurrency.Session) {
 			logutil.BgLogger().Error("recover panic", zap.String("prompt", m.prompt), zap.Any("error", r), zap.String("buffer", string(buf)))
 			metrics.PanicCounter.WithLabelValues(metrics.LabelDDLOwner).Inc()
 		}
+		m.wg.Done()
```

Comment:
If we call `ownerManager.Cancel()`, it seems that we will call `m.wg.Done()` twice.